sequencer/src/block.rs

```diff
@@ -30,13 +30,27 @@ struct FeeReceipt {
     recipient: u64, // FIXME `Address`
     amount: u64,    // FIXME `U256`
 }
+/// Fetch fee receitps from l1. Currently a mock function to be
+/// implemented in the future.
+fn fetch_fee_receipts(_parent: &Header) -> Vec<FeeReceipt> {
+    Vec::from([FeeReceipt::default()])
+}
 
 // TODO use U256. We need to impl `ark_serialize::CanonicalDeserialize`
 // ethers::types::U256
 pub type FeeMerkleTree = UniversalMerkleTree<u64, Sha3Digest, u64, U3, Sha3Node>;
+pub type FeeMerkleCommtment = <FeeMerkleTree as MerkleTreeScheme>::Commitment;
 
-fn fetch_fee_receipts(_parent: &Header) -> Vec<FeeReceipt> {
-    Vec::from([FeeReceipt::default()])
+#[derive(Hash, Clone, Debug, Deserialize, Serialize, PartialEq, Eq)]
+pub struct ValidatedState {
+    /// Root Commitment of Block Merkle Tree
+    pub block_merkle_tree_root: BlockMerkleCommitment,
```

Comment:
We want to keep these roots in the header itself...since ValidatedState will eventually be moved out of the header but we want the roots in there

Reply:
@tbro can you move these roots back to the header proper, and add them to the `impl Committable for Header`? After that I think we can merge this PR

Reply:
Will do.